utils/bazel/llvm-project-overlay/clang/BUILD.bazel

```diff
@@ -2512,6 +2523,14 @@ cc_binary(
     ],
 )
 
+genrule(
+    name = "gen_clang_cl",
```

Comment:
We already have a `binary_alias` macro available with `load("//llvm:binary_alias.bzl", "binary_alias")
`, can you use that? e.g. https://github.com/llvm/llvm-project/blob/64c2156d8802b0d7724f65ce854844670e4ec457/utils/bazel/llvm-project-overlay/llvm/BUILD.bazel#L4439